Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+After Gluster upgrade, it is important that op-version is updated in all peers to avoid any 
+compatibility issues.
+
+###Updating op-version
+
+Once Gluster is upgraded, ensure that the op-version is also updated.
+
+    [root@~]#cat /var/lib/glusterd/vols/tv1/info  | grep ^op-version
+
+
+If this is not done automatically set op-version as below.
+For example, after upgrading to glusterfs-3.7.1, set op-version as:
+
+    [root@~]#gluster volume set all cluster.op-version 30701
+
+### Understanding op-version
+
+The op-version for a given X.Y.Z release will be an integer XYZ, with
```

Comment:
Given the op-version is only incremented if a new feature/volume option is added in the release, I am still expecting a link to the metric I proposed, otherwise users would not even get to know what op-version they have to bump up.